.github/workflows/sync_labels.yaml

```diff
@@ -0,0 +1,35 @@
+name: Sync labels
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled]
+    branches: [master, next]
+  issues:
+    types: [labeled, unlabeled]
+
+jobs:
+  label-sync:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      issues: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Fetch all history for all tags and branches
+
+      - name: Install dependencies
+        run: sudo apt-get install -y python3-github
+
+      - name: Run python script for pull request
+        if: ${{ github.event_name == 'pull_request' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: python .github/workflows/sync_labels.py --repository ${{ github.repository }} --number ${{ github.event.number }} --label_action ${{ github.event.action }}
```

Comment:
`.github/workflows/sync_labels.py` -> `.github/scripts/sync_labels.py`
`sync_labels.py` script located under `scripts` folder

Reply:
Oh, forgot to change it back from my debug session. Thanks